src/plugins/discover/public/application/view_components/canvas/top_nav.tsx

```diff
@@ -104,6 +107,16 @@ export const TopNav = ({ opts, showSaveQuery }: TopNavProps) => {
       onQuerySubmit={opts.onQuerySubmit}
       savedQueryId={state.savedQuery}
       onSavedQueryIdChange={updateSavedQueryId}
+      showDataSourceMenu={dataSource?.dataSourceEnabled}
```

Comment:
Im a lil confused how this PR works. If you are using the local cluster, whats the purpose of this menu?